evm/governance.js

```diff
@@ -231,6 +257,67 @@ async function processCommand(options, chain) {
             break;
         }
 
+        case 'gatewayUpgrade': {
+            if (contractName === 'AxelarServiceGovernance') {
+                throw new Error(`Invalid governance action for AxelarServiceGovernance: ${governanceAction}`);
+            }
+
+            if (unixEta < getCurrentTimeInSeconds() + contractConfig.minimumTimeDelay && !yes) {
+                printWarn(`${eta} is less than the minimum eta.`);
+                const answer = readlineSync.question(`Proceed with ${governanceAction}?`);
+                if (answer !== 'y') return;
+            }
+
+            if (!isAddress(implementation) || implementation === AddressZero) {
```

Comment:
```suggestion
            if (!isAddress(implementation) || implementation === AddressZero) {
```
Make the AddressZero check inside `isAddress` itself? Maybe `isAddress` can take an optional value to allow zero address in rare cases when that needs to be allowed, but defaults to false.